src/hypothesis/extra/django/models.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, model, mappings):
             strategy=st.fixed_dictionaries(mappings))
 
     def __repr__(self):
-        return u'ModelStrategy(%s)' % (self.model.__name__,)
+        return 'ModelStrategy(%s)' % (self.model.__name__,)
```

Comment:
Apparently what I mean is that I've got a misconception about reprs in Python 2. Sorry, my error not yours.